packages/app/navigation/GroupSettingsStack.tsx

```diff
@@ -10,10 +14,37 @@ import { GroupSettingsStackParamList } from './types';
 
 const GroupSettings = createNativeStackNavigator<GroupSettingsStackParamList>();
 
-export function GroupSettingsStack() {
+type GroupMetaProps = {
+  navigation: NativeStackNavigationProp<
+    GroupSettingsStackParamList,
+    'GroupMeta'
+  >;
+  route: {
+    key: string;
+    name: 'GroupMeta';
+    params: {
+      groupId: string;
+    };
+  };
+};
+
+export function GroupSettingsStack({
+  navigation,
+}: {
+  navigation: NavigationProp<GroupSettingsStackParamList>;
+}) {
+  const navigateToHome = () => {
+    navigation.navigate('ChatList' as never);
+  };
+
   return (
     <GroupSettings.Navigator screenOptions={{ headerShown: false }}>
-      <GroupSettings.Screen name="GroupMeta" component={GroupMetaScreen} />
+      <GroupSettings.Screen
+        name="GroupMeta"
+        component={(props: GroupMetaProps) => (
+          <GroupMetaScreen {...props} navigateToHome={navigateToHome} />
+        )}
+      />
```

Comment:
```suggestion
      <GroupSettings.Screen name="GroupMeta">
        {(props: GroupMetaProps) => (
          <GroupMetaScreen {...props} navigateToHome={navigateToHome} />
        )}
      </GroupSettings.Screen>
```

You can just pass it like this.

Reply:
Today I learned. Thank you